physionet-django/notification/utility.py

```diff
@@ -1025,3 +1025,19 @@ def notify_event_participant_application(request, user, registered_user, event):
     body = loader.render_to_string('events/email/event_registration.html', context)
     # Not resend the email if there was an integrity error
     send_mail(subject, body, settings.DEFAULT_FROM_EMAIL, [user.email], fail_silently=False)
+
+
+def notify_users_of_training_expiry(user, training, expiry):
```

Comment:
I see, so the idea is that the training expires after a fixed period, and then it needs to be retaken?

Reply:
Yes, the training would expire and users would be unable to access the projects/data and would need to retake the training.